teuthology/orchestra/daemon/systemd.py

```diff
@@ -95,7 +95,11 @@ def parse_line(line):
     @property
     def pid(self):
         proc_name = 'ceph-%s' % self.type_
-        proc_regex = '"%s.*--id %s"' % (proc_name, self.id_)
+        proc_regex = '"%s.*--id %s "' % (proc_name, self.id_)
+
+        if self.type_ == "rgw":
+            proc_regex = "{}.*--name.*{}".format(self.daemon_type, self.id_)
```

Comment:
Could you please add here a comment which kind of string it is trying to match in the source code, 
also maybe make regex a less cryptic, like put spaces around `--name`,  and use something else rather then `self.daemon_type`.

Reply:
@kshtsk , added comments for process regex match and updated regex. `self.daemon_type` provides `radogw` for rgw daemon, makes easy to constitute rgw process regex.